src/test/java/nl/tudelft/ewi/devhub/webtests/AssignmentTest.java

```diff
@@ -28,5 +29,25 @@ public void testListAssignments() {
 		assertEquals(assignment.getName(), "Product vision");
 		assertEquals(assignment.getNumber(), "1");
 	}
+
+	@Test
+	public void testAssignment() throws ParseException {
+		final String net_id = "assistant1";
```

Comment:
Please store these values in a static variable, such as `ASSISTANT_USERNAME` :smile: